### PR TITLE
Backup file rotation for 30 days

### PR DESCRIPTION
**Pull request by liam-lai**
#### Description
Backup file rotation for 30 days

#### Checklist
_Put an `x` in the boxes that apply. This is simply a reminder of what we are going to look for before merging your code._

- [x] Deployed to QA env successfully
- [x] Tested the feature on QA env with the expected behaviours
- [ ] Tested on all supported browsers (if it's an UI task)
- [ ] Checked no runtime error or warning in the browser console (if it's an UI task)
- [ ] Included an automated test (always better to have!)

### CODE REVIEW

**Comment by josephj** [comment]
@liam-lai 

The code is totally fine for me. The thing I want to verify is the log and the backup.

`vim /var/log/monit` Can't find anything related to `DatabaseUtil#backupDatabase`.

**Comment by liam-lai** [comment]
@josephj 
In the log dir: 
```
[ec2-user@ip-10-10-3-42 log]$ pwd
/opt/efcsydney-roster/log
[ec2-user@ip-10-10-3-42 log]$ grep Remove log/2018-04-02-log.log
{"level":"info","message":"[END] DatabaseUtil#backupDatabase; Remove file: efcRosterQa_20180304.sql, at: Tue Apr 03 2018 00:00:00 GMT+1000","timestamp":"00:00:00"}
```

**Comment by josephj** [comment]
Looks good!

File: api/utilities/database-util.js [hunk 1]
```diff
@@ -8,12 +8,16 @@ const config = require('config');
...
 
-function getDateString() {
+function getCurrentDateString() {
   return moment().format('YYYYMMDD');
 }
 
-function getBackupFileName() {
-  return `${dbConfig.database}_${getDateString()}.sql`;
+function getLastMonthString() {
+  return moment().subtract(30, 'days').format('YYYYMMDD');
```
Comment (yhua045) [inline api/utilities/database-util.js]: can you subtract by month? like subtract(1, 'month')? February has only 28 days if we subtract by days, that may give you an unexpected result?

Comment (liam-lai) [reply api/utilities/database-util.js]: If we subtract by month, then some date may not be able to select. For example 20180131.
```
var moment = require('moment');
console.log(moment('20180301').subtract(1, 'months').format('YYYYMMDD')); 
//20180201
console.log(moment('20180228').subtract(1, 'months').format('YYYYMMDD')); 
//20180128
console.log(moment('20180227').subtract(1, 'months').format('YYYYMMDD'));
//20180127

console.log(moment('20180331').subtract(1, 'months').format('YYYYMMDD')); 
//20180228
console.log(moment('20180330').subtract(1, 'months').format('YYYYMMDD'));
//20180228
console.log(moment('20180329').subtract(1, 'months').format('YYYYMMDD'));
//20180228
```

**Pull request merged**